site/public/js/forum.js

```diff
@@ -1358,8 +1358,6 @@ function thread_post_handler(){
     $('.post_reply_from').submit(function(){
```

Comment:
this whole function is for disabling the submit button, the first two lines select the elements while the next two lines apply the disabling so it should be removed (lines 1358 : 1362)